### DISCUSSION TITLE
Getting 404 when checking for account balance

### DISCUSSION THREAD

**Discussion by NJORO45** [njoguamos/laravel-jenga#49] [General]
`public static function signaturetest($token) {
        $privateKeyPath = "C:/Users/hp/privatekey.pem"; 
        $plainText  = "0582910862KE2018-08-09"; // Correct format for signing
        $privateKey = openssl_pkey_get_private(file_get_contents($privateKeyPath));
    
        openssl_sign($plainText, $signature, $privateKey, OPENSSL_ALGO_SHA256);
    
        $curl = curl_init();
        $data_string = json_encode([
            "countryCode" => "KE",
            "accountId" => "0582910862",//my account 
            "date" => "2018-08-09"
        ]);
    
        curl_setopt_array($curl, [
            CURLOPT_URL => "https://uat.finserve.africa/v3-apis/account-api/v3.0/accounts/balances/KE/0120284450362", // Ensure this is the correct endpoint
            CURLOPT_RETURNTRANSFER => true,
            CURLOPT_ENCODING => "",
            CURLOPT_MAXREDIRS => 10,
            CURLOPT_TIMEOUT => 30,
            CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
            CURLOPT_CUSTOMREQUEST => "POST",
            CURLOPT_POSTFIELDS => $data_string,
            CURLOPT_HTTPHEADER => [
                "Authorization: Bearer " . $token,
                "Content-Type: application/json",
                "signature: " . base64_encode($signature)
            ]
        ]);
        
        $result = curl_exec($curl);
        $err = curl_error($curl);
    
        curl_close($curl);
    
        if ($err) {
            echo "cURL Error #:" . $err;
        } else {
            echo $result;
        }
    }`

**Comment by njoguamos** [comment]
### Here are some of the suggestions you can consider:

1. Ensure you follow the [API documentation](https://developer.jengahq.io/api-explorer/account-services/account-balance). E.g.g the endpoint does not seem to take any `body params`, yet you have included

2. Ensure that you have enabled balance inquiry from the Jenga API dashboard. If you have not enabled it, you will not be able to access the endpoint.

3. I also suggest you use a live URL where you test a real bank account that you have linked in Jenga API.
